### DISCUSSION TITLE
Email

### DISCUSSION THREAD

**Discussion by KhaosTheMaxim** [firefly-iii/firefly-iii#7785] [Firefly III]
Whenever it attempts the test email it says connect time out on the email's smtp address.

**Comment by JC5** [comment]
It would seem the address and port combination is incorrect.

**Reply by KhaosTheMaxim** [reply]
I went over it many times and it is correct. It is the same info I used for other services. I tried use curl to ifconfig.io to see if there was a response but it would also time out.

**Reply by JC5** [reply]
I'm not sure what it could be. There's no magic in the container, it's basically the standard PHP container with Firefly III on top.
